myokit/formats/sbml/_importer.py

```diff
@@ -371,39 +1103,43 @@ class SBMLError(myokit.ImportError):
     """
 
 
-unit_map = {
-    'dimensionless': myokit.units.dimensionless,
+# SBML base units according to libSBML
+SBML2MyoKitUnitDict = {
     'ampere': myokit.units.A,
-    'farad': myokit.units.F,
-    'katal': myokit.units.kat,
-    'lux': myokit.units.lux,
-    'pascal': myokit.units.Pa,
-    'tesla': myokit.units.T,
+    'avogadro': None,  # TODO: myokit equivalent not yet looked up
     'becquerel': myokit.units.Bq,
-    'gram': myokit.units.g,
-    'kelvin': myokit.units.K,
-    'meter': myokit.units.m,
-    'radian': myokit.units.rad,
-    'volt': myokit.units.V,
     'candela': myokit.units.cd,
-    'gray': myokit.units.Gy,
-    'kilogram': myokit.units.kg,
-    'metre': myokit.units.m,
-    'second': myokit.units.s,
-    'watt': myokit.units.W,
     'celsius': myokit.units.C,
-    'henry': myokit.units.H,
-    'liter': myokit.units.L,
-    'mole': myokit.units.mol,
-    'siemens': myokit.units.S,
-    'weber': myokit.units.Wb,
     'coulomb': myokit.units.C,
+    'dimensionless': myokit.units.dimensionless,
+    'farad': myokit.units.F,
+    'gram': myokit.units.g,
+    'gray': myokit.units.Gy,
+    'henry': myokit.units.H,
     'hertz': myokit.units.Hz,
-    'litre': myokit.units.L,
-    'newton': myokit.units.N,
-    'sievert': myokit.units.Sv,
+    'item': None,  # TODO: myokit equivalent not yet looked up
```

Comment:
What kind of unit is `item` ???

Reply:
I have no idea :D the units in SBML lvl3 v2 are defined here

https://livermetabolism.com/paper/sbml-level-3-version-2-core.pdf

on page 44.

Reply:
" “item” is used for expressing such things as “N items” when “mole” is not an appropriate unit"

Reply:
So it seems to be quite generic. Is there an equivalent in myokit?

Reply:
Nah, I'd just map it to dimensionless